cmd/sources-controller/main.go

```diff
@@ -93,6 +96,11 @@ func main() {
 			containerSourceInformer,
 			deploymentInformer,
 		),
+		apiserversource.NewController(
+			opt,
+			apiserverSourceInformer,
+			deploymentInformer,
```

Comment:
Could we just pass the RA Image here instead of buried in the controller?

Reply:
I guess this is a general issue applying to other sources, like `cronjob`